context.go

```diff
@@ -122,7 +135,10 @@ func (c *Context) Mul(d, x, y *Decimal) (Condition, error) {
 }
 
 // Quo sets d to the quotient x/y for y != 0. c's Precision must be > 0.
+// REVIEW: s/c's Precision/c.Precision/
 func (c *Context) Quo(d, x, y *Decimal) (Condition, error) {
+	// REVIEW: should we return an error for Precision < 0
```

Comment:
Oh precision < 0, I misread that is == 0. Precision is a uint32, so it can't be < 0.